src/Application/Executor.cs

```diff
@@ -24,6 +24,7 @@ internal static class Executor
             var translateImage = Environment.GetEnvironmentVariable("SYMBOLICA_TRANSLATE_IMAGE");
 
             File.Delete(Path.Combine(directory, "symbolica.bc"));
+            File.Delete(Path.Combine(directory, ".symbolica.bc"));
```

Comment:
Obviously the json file is gone now and replaced with a "translated" bitcode file. This seemed like a reasonable name for it, but hopefully we manage to do the "translation" internally in memory soon and it goes away again.

Reply:
So just for my own sake. `symbolica.bc` is the output from the "build" stage and then `.symbolica.bc` is the translated version of this after we've run the "translate" docker image over it? I wonder if we should name it a bit differently to make the distinction clearer, even if it is only temporary.

Reply:
We could do. It is just a string in the translate script. Here is simply where we clean it up if it exists from any previous run. I didn't put much thought into it because I am planning to kill it imminently.